src/components/GridGallery.js

```diff
@@ -70,7 +70,7 @@ const GridGallery = ({ thumbs, images, columns, countIndicator, thumbAspect, con
             }
           </CircularCarousel>
           : <div className='mobile-flat'>
-            <div className='mobile-grid-container'>
+            <div className='mobile-grid-container' style={removeGrayBackground ? { background: 'rgb(242,242,242)' } : { background: 'transparent' }}>
```

Comment:
Is this backwards? Remove should set it to transparent no? 
Also, this is a small formatting thing, but maybe use this instead?
`style={{ background: removeGrayBackground ? 'transparent' : 'rgb(242, 242, 242)' }}`

Reply:
This is weird.. that's an older version of my code from the other branch.. must've snuck in there somehow when I was switching branches. Fix is coming